app/src/controllers/log/actionCreators.js

```diff
@@ -59,6 +59,11 @@ export const setPageLoadingAction = (isLoading) => ({
   payload: isLoading,
 });
 
+export const setIncludeAllLaunches = (includeAllLaunches) => ({
```

Comment:
All action creators must have an `Action` postfix, so in this case action will be `setIncludeAllLaunchesAction`